xterm/acl_security.pl

```diff
@@ -10,11 +10,20 @@ sub acl_security_form
 print &ui_table_row($text{'acl_user'},
 	&ui_opt_textbox("user", $o->{'user'} eq '*' ? undef : $o->{'user'},
 			20, $text{'acl_sameuser'}));
+
+if ($o->{'user'} eq "root" && $remote_user ne $o->{'user'}) {
```

Comment:
We should just always show all fields in the ACL, not make them conditional on other fields.

Reply:
Well, this option has nonsense for a regular user, right?

Reply:
Right, but the `$o->{'user'}` field that controls if this field is visible is also editable on the ACL form. So it the admin changes the `user` field from something else to `root` he won't be able to adjust this option without saving and then re-opening the page.

Reply:
Oh, that's right. What about this fix then https://github.com/webmin/webmin/pull/2195/commits/a12f385a5bc9ba1c2c8dc81fe42126546fdaa18c.

Reply:
Just keep it simple and always show that field ... no need for any if statement

Reply:
Well .. okay!